pandas/io/parsers.py

```diff
@@ -40,70 +48,79 @@ class ParserWarning(Warning):
+    multi-index on the columns e.g. [0,1,3]. Intervening rows that are not
+    specified will be skipped (e.g. 2 in this example is skipped). Note that
+    this parameter ignores commented lines and empty lines if
+    ``skip_blank_lines=True``, so header=0 denotes the first line of data rather
+    than the first line of the file.
+names : array-like, default None
+    List of column names to use. If file contains no header row, then you
+    should explicitly pass header=None
 index_col : int or sequence or False, default None
     Column to use as the row labels of the DataFrame. If a sequence is given, a
     MultiIndex is used. If you have a malformed file with delimiters at the end
     of each line, you might consider index_col=False to force pandas to _not_
     use the first column as the index (row names)
-names : array-like, default None
-    List of column names to use. If file contains no header row, then you
-    should explicitly pass header=None
-prefix : string, default None
-    Prefix to add to column numbers when no header, e.g 'X' for X0, X1, ...
-na_values : str, list-like or dict, default None
-    Additional strings to recognize as NA/NaN. If dict passed, specific
-    per-column NA values
+usecols : array-like, default None
+    Return a subset of the columns.
+    Results in much faster parsing time and lower memory usage.
+squeeze : boolean, default False
+    If the parsed data only contains one column then return a Series
+prefix : str, default None
+    Prefix to add to column numbers when no header, e.g. 'X' for X0, X1, ...
+mangle_dupe_cols : boolean, default True
+    Duplicate columns will be specified as 'X.0'...'X.N', rather than 'X'...'X'
+dtype : Type name or dict of column -> type, default None
+    Data type for data or columns. E.g. {'a': np.float64, 'b': np.int32}
+    (Unsupported with engine='python'). Use `str` or `object` to preserve and
+    not interpret dtype.
+%s
+converters : dict, default None
+    Dict of functions for converting values in certain columns. Keys can either
+    be integers or column labels
 true_values : list, default None
     Values to consider as True
 false_values : list, default None
     Values to consider as False
+skipinitialspace : boolean, default False
+    Skip spaces after delimiter.
+skiprows : list-like or integer, default None
+    Line numbers to skip (0-indexed) or number of lines to skip (int)
+    at the start of the file
+skipfooter : int, default 0
+    Number of lines at bottom of file to skip (Unsupported with engine='c')
+nrows : int, default None
+    Number of rows of file to read. Useful for reading pieces of large files
+na_values : str or list-like or dict, default None
+    Additional strings to recognize as NA/NaN. If dict passed, specific
+    per-column NA values.  By default the following values are interpreted as
+    NaN: `'""" + "'`, `'".join(sorted(_NA_VALUES)) + """'`.
```

Comment:
I added this so the list of na_values is always up to date with the code. I'm wondering if it's maybe a bit too fancy though.